lib/motors/can_processor.cpp

```diff
@@ -1,10 +1,28 @@
 #include "can_processor.hpp"
+#include "controller.hpp"
+
+#include <climits>
+#include <cstdint>
+#include <iostream>
 
 namespace hyped::motors {
 
-CanProcessor::CanProcessor()
+uint64_t CanProcessor::little_endian_array_to_big_endian_int(
```

Comment:
This function isn't needed anymore so ditch it and you should pass your std test